pom.xml

```diff
@@ -877,7 +830,7 @@ under the License.
 			</properties>
 			<activation>
 				<property>
-					<name>scala-2.12</name>
+					<name>!scala-2.11</name>
```

Comment:
Even if a user does not specify `-Dscala-2.12`, the 2.12 profile will trigger. I wanted to avoid using `activeByDefault`.